content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -154,7 +153,239 @@ Set the value to `true` to disable data export, e.g. for testing purposes.
 
 {{% /config_option %}}
 
-### Additional instrumentations
+### OTLP Exporter
+
+This package provides auto configuration for the
+[OTLP](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/otlp)
+and
+[Logging](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging)
+Span Exporters.
+
+#### Exporter Properties
+
+| Feature       | Property                      | Default Value  |
+| ------------- | ----------------------------- |----------------|
+| OTLP Exporter | `otel.exporter.otlp.endpoint` | localhost:4317 |
```

Comment:
this effectively changed in 2.0.0 when we changed the default protocol to `http/protobuf` to align with the spec
```suggestion
| OTLP Exporter | `otel.exporter.otlp.endpoint` | http://localhost:4318 |
```

Reply:
@trask: would it be useful to actually encode `http://localhost:4318` using markdown raw-link syntax like this `<http://localhost:4318>`, so that it renders as a link like this <http://localhost:4318>?

Reply:
it looks like we don't over here: https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_endpoint

maybe we could consider referencing those spec sections(?)

Reply:
looks like the linter requires that the link be formatted as `<http://localhost:4318>` so I can update that, but would it be preferable for me to instead remove the table all together and add something like:

```
### OTLP Exporter

This package provides auto configuration for the
[OTLP](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/otlp)
and
[Logging](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging)
Span Exporters.

As of 2.0.0+ the default protocol is http/protobuf, for more details on exporter configuration options see [OTLP Exporter Configuration](/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_endpoint).
```

Reply:
Either works for me. Note that I tweaked your suggested markdown in the previous comment. Also note that the config page that you're linking to might move :) -- see:

- #3888

Reply:
i will watch this PR and can update accordingly when it is merged